### PR TITLE
added new features to homepage

### PR DESCRIPTION
**Pull request by GeorgeMocanu23**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
-import { signIn, useSession } from "next-auth/react"
+import { useSession } from "next-auth/react";
+import { Typography } from '@mui/material';
```
Comment (S7012MY) [inline pages/index.tsx]: we don't use ; in this project

File: public/themes/welcomeTheme.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createTheme } from '@mui/material/styles';
```
Comment (S7012MY) [inline public/themes/welcomeTheme.js]: let's use default.js as the name of the file and theme. Or even better salesWell.js

File: pages/index.tsx [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
-    Welcome to SalesWell{session ? ' ' + session.user.name : ''}!
-    <br />
-    { !session ? 
-      <button onClick={() => signIn()}>Sign in with Email</button> 
-      : null }
-  </div>
+  return (
+    <ThemeProvider theme={ welcomeTheme }>
+      <Typography variant="subtitle1">
+        Welcome{
```
Comment (S7012MY) [inline pages/index.tsx]: Please fix the indentation

File: pages/index.tsx [hunk 4]
```diff
@@ -1,15 +1,20 @@
...
-  return <div>
-    Welcome to SalesWell{session ? ' ' + session.user.name : ''}!
-    <br />
-    { !session ? 
-      <button onClick={() => signIn()}>Sign in with Email</button> 
-      : null }
-  </div>
+  return (
+    <ThemeProvider theme={ welcomeTheme }>
+      <Typography variant="subtitle1">
```
Comment (S7012MY) [inline pages/index.tsx]: can't we use dirrectly h1/h2...?

File: public/themes/welcomeTheme.js (hunk 2 above)
Comment (S7012MY) [reply public/themes/welcomeTheme.js]: Also, are we sure we need to create a theme?

**Review by S7012MY — CHANGES_REQUESTED** [review]

File: pages/index.tsx [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
-    Welcome to SalesWell{session ? ' ' + session.user.name : ''}!
-    <br />
-    { !session ? 
-      <button onClick={() => signIn()}>Sign in with Email</button> 
-      : null }
-  </div>
+  return (
+    <Typography variant="h4" style={{ textAlign: 'center' }}>
+      Welcome
+      { session ? ' back, ' + session.user.name + ' !' : ' to SalesWell' }
```
Comment (keepcoding-tech) [inline pages/index.tsx]: There shouldn't be a space before `!`

**Review by keepcoding-tech — CHANGES_REQUESTED** [review]

**Review by keepcoding-tech — APPROVED** [review]

**Review by S7012MY — APPROVED** [review]

**Pull request merged**